### PR TITLE
Babel decorator plugin URL change

### PR DESCRIPTION
**Pull request by fgm**
It moved from regenerator-* to babel-*

### CODE REVIEW

**Review by ljharb — DISMISSED** [review]

**Comment by ljharb** [comment]
Looks like you need to run the build process

**Comment by fgm** [comment]
Built

**Review by chicoxyzzy — APPROVED** [review]

**Review by afmenez — APPROVED** [review]
LGTM

**Review by ljharb — APPROVED** [review]

File: data-esnext.js [hunk 1]
```diff
@@ -1183,7 +1183,7 @@ exports.tests = [
         return Object.getOwnPropertyDescriptor(A.prototype, "B").configurable === false;
       */},
       res: {
-        babel6: {val: false, note_id: "regenerator-decorators-legacy", note_html: "Babel 6 still has no official support decorators, but you can use <a href='https://github.com/loganfsmyth/regenerator-plugin-transform-decorators-legacy'>this plugin</a>."},
+        babel6: {val: false, note_id: "regenerator-decorators-legacy", note_html: "Babel 6 still has no official support decorators, but you can use <a href='https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy'>this plugin</a>."},
```
Comment (zloirock) [inline data-esnext.js:1186]: I don't know why `regenerator` was mentioned here. Also, it's still in `note_id`.

Comment (chicoxyzzy) [reply data-esnext.js:1186]: Good catch! We definitely should fix `note_id`.

**Comment by fgm** [comment]
Since I'm on it, #1271 contains that other change.

**Pull request merged**